### ISSUE TITLE
Validation loss is uninformative

### ISSUE THREAD

**Issue by NoahHayes17** [paninski-lab/beast#59]
## Summary
During validation, the rendered image is composited on a white background while the ground-truth target is composited on black. The validation loss includes an unmasked perceptual term over the whole image, so it's dominated by this white-vs-black mismatch as the background is most of the frame. As a result `val_loss` and `val_perceptual` are inflated to a near-constant value that doesn't track reconstruction quality (`val_psnr` is unaffected as it comes from the *masked* L2 term).

## Cause
1. `beast/models/beast3d/beast3d_model.py`: `_sample_background()` returns `None` outside training:
   ```python
   if self.training and self.random_background:
       return torch.rand(3, device=device, dtype=dtype)
   return None
   ```
2. `beast/rendering/gaussians_renderer.py`: `render_opencv_cam_gsplat` turns that `None` into white:
   ```python
   if bg_color is None:
       bg_color = (1.0, 1.0, 1.0)
   ```
3. `beast/models/beast3d/beast3d_model.py`: `_prepare_target` puts the target on black (the `bg_color` composite is skipped when it's `None`):
   ```python
   target = target_image * mask_fg
   if bg_color is not None:
       target = target + bg_color.view(1, 1, 3, 1, 1) * (1.0 - mask_fg)
   ```
4. `beast/models/erayzer/erayzer_model.py`: the L2 term is masked (fine), but the perceptual term is unmasked and part of the total loss:
   ```python
   l2_loss = masked_mse_loss(rendering, target, pixel_mask)
   perceptual_loss = self.perceptual_loss_module(rendering, target)
   total_loss = ... + perceptual_loss_weight * perceptual_loss
   ```

## Suggested fix
Return a fixed background during validation so the render and the composited target share it (`bg_color` is already threaded to both):
```python
if self.training and self.random_background:
    return torch.rand(3, device=device, dtype=dtype)
return torch.zeros(3, device=device, dtype=dtype)
```
